### ISSUE TITLE
Provide scraper for Azure DataBricks

### ISSUE THREAD

**Issue by gburton1** [tomkerkhove/promitor#2117]
### Proposal

@tomkerkhove do you have any thoughts on whether there could be a generalized, viable way to monitor Azure DataBricks with Promitor? If there was a clean pathway, we could contribute. Azure officially offers this approach: https://docs.microsoft.com/en-us/azure/architecture/databricks-monitoring/

We are exploring alternatives as well, such as exposing Prometheus metrics directly on the Spark nodes.

### Component

Scraper

**Comment by tomkerkhove** [comment]
Unfortunately it does not look like Databricks is providing metrics in Azure Monitor - https://docs.microsoft.com/en-us/azure/azure-monitor/essentials/metrics-supported

**Comment by locmai** [comment]
Yes, Databricks service is using the new way to export the metrics via Azure Log Analytics. This might help - https://github.com/tomkerkhove/promitor/issues/1076 

Related discussion: https://github.com/tomkerkhove/promitor/discussions/1975
